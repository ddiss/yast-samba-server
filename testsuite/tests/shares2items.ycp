{

// test for shares2items conversion
// $Id$

    include "testsuite.ycp";

    TESTSUITE_INIT([
            $[
                "target" : $[
                    "tmpdir" : "/tmp"
                ]
            ]
        ],nil);
    
    include "samba-server/routines.ycp";

    TEST( ``(shares2items( $[ "homes" : $[ "path": "/bla", "comment": "Haha", "commentout": false], 
	"netlogon" : $[ "path": "/bla", "comment": "BooHaha", "commentout": true]
    ] ) ), [], nil );    
}
