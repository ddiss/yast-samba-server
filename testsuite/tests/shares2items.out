Read	.target.size "/etc/install.inf" nil
Read	.etc.install_inf.Sourcemounted nil
Read	.etc.install_inf.AutoYaST nil
Read	.etc.install_inf.Repair nil
Read	.etc.install_inf.Textmode nil
Read	.etc.install_inf.Manual nil
Read	.target.string "/proc/cmdline" nil
Read	.etc.install_inf.Console nil
Read	.etc.install_inf.Braille nil
Read	.etc.install_inf.VNC nil
Read	.etc.install_inf.UseSSH nil
Read	.etc.install_inf.InstMode nil
Return	[`item (`id (1), "Enabled", "homes", "/bla", "Haha"), `item (`id (2), "Disabled", "netlogon", "/bla", "BooHaha")]
